### DISCUSSION TITLE
Demo support

### DISCUSSION THREAD

**Discussion by nht9499** [retextjs/retext#91] [Q&A]
import {retext} from 'retext'
import retextPos from 'retext-pos'

Where can I install these packages, I can't found it in readme. Please let me know

**Comment by ChristianMurphy** [comment]
Welcome @nht9499!
There are several ways to install, they are listed in the readme of each project under the section labelled "install".
https://github.com/retextjs/retext/tree/main/packages/retext#install
https://github.com/retextjs/retext-pos#install
Give them a try in the project directory where you want to use `retext`.
